build/build.cake

```diff
@@ -150,7 +150,8 @@ Task("BuildProjects")
     Information("\nBuilding Solution");
     var buildSettings = new MSBuildSettings
     {
-        MaxCpuCount = 0
+        MaxCpuCount = 0,
+        PlatformTarget = PlatformTarget.MSIL
```

Comment:
Is there a `Platform = AnyCPU` equivalent?

Wouldn't this override inference based on `Platform`?

Reply:
This is the equivalent of Any CPU. Their docs states:
`Platform target: MSIL (Any CPU)`
https://cakebuild.net/api/Cake.Common.Tools.MSBuild/PlatformTarget/4F91A79B